src/app/reducers.js

```diff
@@ -0,0 +1,29 @@
+// reducer for state.metagraph
+export function metagraph(state = {}, action) {
+  switch (action.type) {
```

Comment:
I'm not sure if we'll ever have more cases. In this situation, I'm using `switch`/`case` because it seems to be the expected pattern for reducers in redux. So anyone viewing my code in the future would see that and understand that it's a reducer.

Also, since some of the other reducers use `switch`/`case`, I'd like to keep them all consistent.